NavigationReactNative/src/ios/NVNavigationBarView.m

```diff
@@ -18,6 +18,16 @@ - (void)didSetProps:(NSArray<NSString *> *)changedProps
     [self.reactViewController.navigationController setNavigationBarHidden:self.hidden];
     if ([changedProps containsObject:@"title"]) {
         [self.reactViewController.navigationItem setTitle:self.title];
+    } else if ([changedProps containsObject:@"backTitle"]) {
```

Comment:
This should be `if` instead of an `else if` because you can change both the `title` and the `backTitle` props at the same time